ui/raidboss/data/05-shb/raid/e2s.js

```diff
@@ -356,32 +269,21 @@
       condition: function(data, matches) {
         return data.me == matches.target && data.spell[data.me] == 'fire';
       },
-      alertText: {
-        en: 'Spread',
-        de: 'Verteilen',
-        fr: 'Dispersez-vous',
-        ja: '散開',
-        cn: '散开',
-        ko: '산개',
-      },
+      response: Responses.spread('alert'),
     },
     {
-      id: 'E2S Shadoweye No Waiting',
+      id: 'E2S Shadoweye No Waiting LookAwayFrom',
```

Comment:
Same as above, can this be `E2S Shadoweye No Waiting` and `E2S Shadoweye No Waiting You`